public/js/Constants.js

```diff
@@ -34,6 +44,28 @@ var Constants: {
   resourceSpawnRate: 60,
 
   maxHistoryBuffer: 500,
+
+  animals: ['alligator', 'ant', 'antelope', 'bat', 'bear', 'beaver', 'bee', 'beetle',
```

Comment:
would be better if you can figure out a way to pull this from a separate text file but if it takes too much effort, don't worry about it not a big deal